WeScanSampleProject/HomeViewController.swift

```diff
@@ -29,11 +29,11 @@ final class HomeViewController: UIViewController {
     
     lazy private var scanButton: UIButton = {
         let button = UIButton(type: .custom)
-        button.setTitle("Scan Now!", for: .normal)
+        button.setTitle("Scan", for: .normal)
```

Comment:
Why did you decide to change the copy and styling of the button?

Reply:
I felt it was better this way - but I can revert it if you want.